db/migrate/20170808180359_add_timestamp_rent.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimestampRent < ActiveRecord::Migration[5.1]
+  def change
+    add_timestamps :rents, default: DateTime.now
```

Comment:
Remove the `defautl: ...` Timestamps should't have a default, Rails will populate those fields